include/roboteam_ai/world_new/Team.hpp

```diff
@@ -0,0 +1,25 @@
+//
+// Created by john on 12/17/19.
+//
+
+#ifndef RTT_TEAM_HPP
+#define RTT_TEAM_HPP
+
+namespace rtt::world_new {
+    /**
+     * Enum used for indicating team
+     */
+    enum Team : short {
+        // our team
+        us,
+        // their team
+        them,
+        /**
+         * If a robot has both as team -> invalid.
+         */
+        both
```

Comment:
does this ever happen?

Reply:
Yeah, getRobotsWithId(3, Team::both)

Also the default invalid == both